sql/config_test.go

```diff
@@ -34,12 +34,8 @@ import (
 
 func getLatestConfig(s *server.TestServer, expected int) (cfg *config.SystemConfig, err error) {
 	err = util.IsTrueWithin(func() bool {
-		var err2 error
-		cfg, err2 = s.Gossip().GetSystemConfig()
-		if err2 != nil {
-			return false
-		}
-		return len(cfg.Values) != expected
```

Comment:
I had this the wrong way around before. The reason it was working was that I also had the wrong value for `expected`.